dftimewolf/cli/recipes/grr_artifact_hosts.py

```diff
@@ -44,16 +45,12 @@
     }],
 }
 
-DEFAULT_ARTIFACT_LIST = (
-    'AllUsersShellHistory,BrowserHistory,'
-    'LinuxLogFiles,AllLinuxScheduleFiles,'
-    'LinuxScheduleFiles,ZeitgeistDatabase,'
-    'AllShellConfigs')
-
 args = [
     ('hosts', 'Comma-separated list of hosts to process', None),
     ('reason', 'Reason for collection', None),
-    ('--artifact_list', 'Comma-separated list of artifacts to fetch',
-     DEFAULT_ARTIFACT_LIST),
+    ('--artifact_list', 'Comma-separated list of artifacts to fetch '
+     '(override default artifacts)', None),
+    ('--extra_artifacts', 'Comma-separated list of artifacts to fetch '
```

Comment:
Maybe: "Comma-separated list of artifacts to append to the default artifact list"?